packages/manager/modules/pci/src/projects/project/private-registry/create/create.controller.js

```diff
@@ -0,0 +1,42 @@
+import get from 'lodash/get';
+
+export default class {
+  /* @ngInject */
+  constructor($stateParams, $translate, CucControllerHelper, privateRegistryService) {
+    this.$translate = $translate;
+    this.cucControllerHelper = CucControllerHelper;
+    this.privateRegistryService = privateRegistryService;
+    this.fromState = $stateParams.fromState;
+
+    this.projectId = $stateParams.projectId;
```

Comment:
If you add `projectId: '<'` in the component's bindings, the `projectId` will be available in the controller with `this.projectId`. You don't need to resolve it for this state as it is already resolved by the `pci.projects.project` state (https://github.com/ovh-ux/manager/blob/master/packages/manager/modules/pci/src/projects/project/project.routing.js#L36), and it is accessible for all child states.